app/assets/stylesheets/admin/attachment.scss

```diff
@@ -3,27 +3,33 @@
 // You can use Sass (SCSS) here: https://sass-lang.com/
 @import "font-awesome";
 
-$border-color: #003087;
+
 $invalid-color: red;
-$background-color: #002855;
+
+thead{
+  position: relative;
+}
 
 .table-attachment {
-  width: 80%;
+  width: 60%;
   height: 60px;
-  border-collapse: collapse;
+  // border-collapse: collapse;
   margin: 0 auto;
   text-align: center;
-  background-color: #F5F5F5;
+  // background-color: $color-table-background;
 }
 
 .title-attachment {
   padding: 0.5rem;
-  border: 1px solid #999;
+  // border: 1px solid $gray-color;
+  text-align: center;
 }
 
-.title {
-  margin-left: 20px;
-  color: grey;
+.title-attachment-text {
```

Comment:
just to be sure - read how you can put nested CSS styles inside SCSS files, it could be useful